README.MD

```diff
@@ -51,8 +51,8 @@ Email Validation Regex
 #### Security
 
 - helmet
-- cors
-- xss-clean
+- cors - cross origin resource sharing. it is mechanism to allow or restrict request resources on web server depending on where HTTP request was initiated.
```

Comment:
Notably, this is _really_ enforced by the browser, not the web server. A machine client can say that the request came from wherever they want; there's no way for the server to definitively know. However, because all the popular web browsers enforce these CORS rules, it is an effective mechanism for controlling how websites will behave when actual people are using them.